src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -693,6 +693,25 @@ record C(int X, int X)
             };
             AssertEx.Equal(expectedMembers,
                 comp.GetMember<NamedTypeSymbol>("C").GetMembers().OfType<PropertySymbol>().ToTestDisplayStrings());
+
+            var expectedMemberNames = new[] {
+                ".ctor",
+                "get_EqualityContract",
+                "EqualityContract",
+                "<X>k__BackingField",
+                "get_X",
+                "set_X",
+                "X",
+                "ToString",
+                "PrintMembers",
+                "op_Inequality",
+                "op_Equality",
+                "GetHashCode",
+                "Equals",
+                "<Clone>$",
+                "Deconstruct"
+            };
+            AssertEx.Equal(expectedMemberNames, comp.GetMember<NamedTypeSymbol>("C").GetPublicSymbol().MemberNames);
```

Comment:
is order guaranteed here?  should this test sort, just to be sfe against that?

Reply:
We probably could relax the order checking if we feel the need.